frontend/src/service/timer-api-service.js

```diff
@@ -1,11 +1,21 @@
 import axios from "axios";
 
-export const postMinutes = minutes => {
-
+export const runPump = minutes => {
     const timer = {
-        minutes: minutes
+        minutes: minutes,
+        stopPump: false
     }
+    return axios
+        .post("/api/timer", timer)
+        .then(response => response.data)
+        .catch(error => console.log(error))
```

Comment:
Just keep in mind: If you catch the error here and don't return anything, but whatever handles your response.data you might run in to difficulties once you get an error because of variables set do undefined